generator/src/main/java/com/linkedin/pegasus/generator/TemplateSpecGenerator.java

```diff
@@ -155,6 +162,16 @@ public Collection<ClassTemplateSpec> getGeneratedSpecs()
     return _classTemplateSpecs;
   }
 
+  /**
+   * Set the regex name patterns for schemas which needs to skip deprecated fields when generating the specs
+   *
+   * @param skipDeprecatedPatterns list of regex name patterns to be set. They will be used to match full name of schemas.
+   */
+  public void setSkipDeprecatedPatterns(@Nonnull List<Pattern> skipDeprecatedPatterns) {
```

Comment:
Rename to be more clear `setSkipDeprecatedFieldPatterns` and `_skipDeprecatedFieldPatterns`.